setup.py

```diff
@@ -0,0 +1,41 @@
+from distutils.core import setup
+from setuptools import setup, Extension, distutils, Command, find_packages
+import setuptools.command.install
+import distutils.command.build
+import distutils.command.clean
+import os
+import shutil
+
+## extending the install command functionality.
+class install(setuptools.command.install.install):
+    def run(self):
+        print ("INFO: Installing hipify_torch")
+        setuptools.command.install.install.run(self)
+        print ("OK: Successfully installed hipify_torch")
+
+## extending the clean command functionality.
+class clean(distutils.command.clean.clean):
+    def run(self):
+        print ("INFO: cleaning hipify_torch")
+        cwd = os.getcwd()
+        build_dir = os.path.join(cwd, "build")
```

Comment:
I'm not sure I understand why we need this to be explicitly done. Doesn't distutils clean remove the build directory? There's some mention of an `-all` option here: https://docs.python.org/3/distutils/apiref.html#module-distutils.command.clean?